run/action.yaml

```diff
@@ -1,41 +1,37 @@
-name: "Sandard Action Runner"
+name: "Standard Action Runner"
 description: "Build a target and run its action."
 
 inputs:
-  github_pat:
-    description: "GitHub token for accessing private repos."
-    required: false
-    default: ${{ github.token }}
-  nix_key:
-    description: "Secret key used for Nix package signing."
-    required: false
-  cache:
-    description: "Nix cache to optionally upload artifacts to"
-    # auto refers to the local nix store, so nothing is uploaded by default
-    default: auto
-    required: false
-  builder:
-    description: "Nix store to build on."
-    default: auto
+  discovery_json:
+    description: "JSON describing the action matrix, found in the discovery phase."
     required: true
-  json:
-    description: "JSON describing the action, found in the discovery phase."
-    required: true
-  nix_ssh_key:
-    description: "SSH private key to send builds to a remote Nix builder."
-    default: none
+
+  nix_signing_key:
+    description: "Signing key used by Nix for package signing."
     required: false
-  ssh_known_hosts:
-    description: "SSH known host keys for Nix builders"
-    default: none
+  nix_cache:
+    description: "Nix cache to optionally cache (potentially signed) build results."
+    default: auto # auto refers to the local nix store, so nothing is uploaded by default
     required: false
   extra_nix_config:
     description: "Configuration to append to the nix.conf."
     required: false
-  discovery_ssh:
-    description: "Nix store URI for a dedicated discovery machine."
+
+  # Authentication c/o Nix for additional private repositories
+  nix_access_tokens:
```

Comment:
Is a new option necessary if we already have extra_nix_config?

Reply:
Hm, it's the previous `github_pat`, but not specific to github...

Maybe it is one of the more common sestitngs, though. Which likely was the motivation behind `github_pat`?

Reply:
We could just default access-tokens to the default token, and if users need to use a more capable token, they can just set the `access-tokens` in the `extra_nix_config`. IIRC Nix just overwrites any conflicting values, so I think it would work.

But I guess this isn't bad either.

Reply:
Yeah, let's leave this expose. It's relatively common and if we can safe people the round-trip over the Nix documentation, I guess that can only be a good thing. (maybe it's just a wish in vanity, anyways :smile: )